roles/openshift_node/defaults/main.yml

```diff
@@ -79,6 +79,8 @@ r_openshift_node_os_firewall_allow:
 - service: Kubernetes service NodePort UDP
   port: "{{ openshift_node_port_range | default('') }}/udp"
   cond: "{{ openshift_node_port_range is defined }}"
+# Allow multiple port ranges to be added to the role
+r_openshift_node_os_firewall_allow: "{{ default_r_openshift_node_os_firewall_allow + (openshift_node_open_ports | default([])) }}"
```

Comment:
I think the ansible filter for this is `union`: http://docs.ansible.com/ansible/latest/playbooks_filters.html#set-theory-filters
```
r_openshift_node_os_firewall_allow: "{{ default_r_openshift_node_os_firewall_allow | union((openshift_node_open_ports | default([]))) }}"
```

Reply:
Union would remove duplicates if they exist, although that wouldn't actually help for this if the condition values were different.  I can't think of any place where we would want duplicates though.